dev/test-studio/form/FormField.tsx

```diff
@@ -0,0 +1,20 @@
+import {EllipsisVerticalIcon} from '@sanity/icons'
+import {Button} from '@sanity/ui'
+import React from 'react'
+import {FieldProps} from 'sanity'
+import {useDocumentPane} from 'sanity/desk'
+
+export function FormField(props: FieldProps) {
+  const {documentType} = useDocumentPane()
+
+  if (documentType === 'fieldActionsTest') {
+    return props.renderDefault({
+      ...props,
+      actions: (
+        <Button fontSize={1} icon={EllipsisVerticalIcon} mode="bleed" padding={2} tabIndex={-1} />
+      ),
+    })
+  }
+
+  return props.renderDefault(props)
+}
```

Comment:
To make it clear that this is only demoing the field actions capability, could we use a different icon and add a description? Currently you have to read the code to understand that an ellipsis button with no action is actually the _intended_ behavior here (I know this is test-studio only, but still provides clarity).

```suggestion
import {ApiIcon} from '@sanity/icons'
import {Box, Button, Text, Tooltip} from '@sanity/ui'
import React from 'react'
import {FieldProps} from 'sanity'
import {useDocumentPane} from 'sanity/desk'

export function FormField(props: FieldProps) {
  const {documentType} = useDocumentPane()

  if (documentType === 'fieldActionsTest') {
    return props.renderDefault({
      ...props,
      actions: (
        <Tooltip
          content={
            <Box padding={2}>
              <Text>Demo: Field actions extension point</Text>
            </Box>
          }
        >
          <Button fontSize={1} icon={ApiIcon} mode="bleed" padding={2} tabIndex={-1} />
        </Tooltip>
      ),
    })
  }

  return props.renderDefault(props)
}
```